Slambook/index.js

```diff
@@ -9,16 +9,46 @@ console.log(form,"form");
 // Add the evenlistner below
 form.addEventListener('submit',function(e){
     e.preventDefault();
-    console.log(document.querySelector('[name="contactName"]').value);
-    console.log(document.querySelector('[name="relationship"]').value);
-    console.log(document.querySelector('[name="likeInMe"]').value);
-    console.log(document.querySelector('[name="hateInMe"]').value);
-    console.log(document.querySelector('[name="deadReaction"]').value);
-    console.log(document.querySelector('[name="firstImpression"]').value);
-    console.log(document.querySelector('[name="beautifulMessage"]').value);
-    console.log(document.querySelector('[name="nickname"]').value);
-    console.log(document.querySelector('[name="songForMe"]').value);
```

Comment:
The function name 'addSpan' does not follow the recommended naming convention. It should be 'addSpanStyle'.